package.json

```diff
@@ -14,7 +14,8 @@
     "@storybook/addons": "^5.2.8",
     "@storybook/react": "^5.2.8",
     "@testing-library/jest-dom": "^5.1.1",
-    "@testing-library/react": "^10.0.4",
+    "@testing-library/react": "^10.4.3",
```

Comment:
Had to update this so that the included version of Dom testing library would be recent enough for user-event